otsserver/rpc.py

```diff
@@ -180,7 +180,15 @@ def do_GET(self):
 
         elif self.path.startswith('/timestamp/'):
             self.get_timestamp()
-
+        elif self.path == '/tip':
+            msg = self.calendar.stamper.unconfirmed_txs[-1].tip_timestamp.msg
+            if msg is not None:
+                self.send_response(200)
+                self.send_header('Content-type', 'application/octet-stream')
+                self.end_headers()
+                self.wfile.write(msg)
+            else:
+                self.send_response(404)
```

Comment:
204 (the server successfully processed the request and is not returning any content) might be a better answer code.

Reply:
204 responses are cachable by default, so I think that does in fact match the semantics we want - the state "no tip right now" is definitely something that makes sense to cache temporarily.

Speaking of, this code should actually be in a `self.get_tip()` function, with appropriate cache control headers set. Similar to the `get_timestamp()` function we already have.